fury/ui.py

```diff
@@ -3349,24 +3355,30 @@ def __init__(self, labels, padding=1, font_size=18,
         position : (float, float)
             Absolute coordinates (x, y) of the lower-left corner of
             the button of the first option.
+        checked_labels: list(string), optional
+            List of labels that are checked on setting up.
         """
+
         self.labels = list(reversed(labels))
         self._padding = padding
         self._font_size = font_size
         self.font_family = font_family
+        self.checked_labels = checked_labels or []
```

Comment:
Any in-place changes to `self.checked_labels` will be reflected to the original `checked_labels` list.